src/types.ts

```diff
@@ -1,6 +1,9 @@
 import { Object } from 'ts-toolbelt';
 import * as ActionCreators from './core/action-creators';
 import { Flow } from './core/flow';
+import * as StorageAPI from './server/db/base';
+
+export { StorageAPI };
```

Comment:
Maybe let's remove Async from the `internal` package since it's now available via StorageAPI?

Reply:
I did consider it, but exposing it via internal does allow someone to use it when not using Typescript.

Reply:
I see, OK.